.gitignore

```diff
@@ -9,7 +9,7 @@
 
 !.golangci.yml
 !/cli/testdata/*.yml
-!/cli/config/testdata/*.yml
+!/config/testdata/*.yml
```

Comment:
`!/cli/config/testdata/*.yml` should be kept AFAICT.

Reply:
Ok (: